commit ab0beafd52b98dfb8b8244b2c6794efbc87478db
Author:     Pablo Neira Ayuso <pablo@netfilter.org>
AuthorDate: Fri Feb 2 10:09:34 2024 +0100
Commit:     Pablo Neira Ayuso <pablo@netfilter.org>
CommitDate: Thu Feb 8 10:27:41 2024 +0100

    netfilter: nft_set_pipapo: remove static in nft_pipapo_get()
    
    This has slipped through when reducing memory footprint for set
    elements, remove it.
    
    Fixes: 9dad402b89e8 ("netfilter: nf_tables: expose opaque set element as struct nft_elem_priv")
    Reported-by: Florian Westphal <fw@strlen.de>
    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>

diff --git a/net/netfilter/nft_set_pipapo.c b/net/netfilter/nft_set_pipapo.c
index efd523496be4..f24ecdaa1c1e 100644
--- a/net/netfilter/nft_set_pipapo.c
+++ b/net/netfilter/nft_set_pipapo.c
@@ -603,7 +603,7 @@ static struct nft_elem_priv *
 nft_pipapo_get(const struct net *net, const struct nft_set *set,
 	       const struct nft_set_elem *elem, unsigned int flags)
 {
-	static struct nft_pipapo_elem *e;
+	struct nft_pipapo_elem *e;
 
 	e = pipapo_get(net, set, (const u8 *)elem->key.val.data,
 		       nft_genmask_cur(net));
